lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -134,7 +134,14 @@ def _mpa_v1(main_script_path: str):
     # Read out the my_pages folder and create a page for every script:
     pages = PAGES_FOLDER.glob("*.py")
     pages = sorted(
-        [page for page in pages if page.name.endswith(".py")], key=page_sort_key
+        [
```

Comment:
(very nit, feel free to ignore): this list comprehension expression is complex enough to maybe rewrite it, although since filter clauses are trivial, it is easy to read and comprehend.

Reply:
For the list comprehension, the two options are:
1. Extract this as a function
2. replace the check with some sort of regex

I think (2) will not help for readability and (1) is a little bit of overkill (I would essentially be showing the same code earlier, so I am opting to keep this.